### ISSUE TITLE
Doesn't work when target is an empty body

### ISSUE THREAD

**Issue by vladshcherbin** [posthtml/posthtml-insert-at#3]
When body is empty and inline, nothing is inserted:

Html:

```html
<!doctype html>
<html>
  <body></body>
</html>
```

Config:

```js
insertAt({
  selector: 'body',
  prepend: '<p>hey</p>',
  append: '<p>ho</p>',
  behavior: 'inside'
})
```

However, when html is changed to not inline, it works:

```html
<!doctype html>
<html>
  <body>
  </body>
</html>
```

Produces correct:

```html
<!doctype html>
<html>
  <body>
    <p>hey</p>
    <p>ho</p>
  </body>
</html>
```

Everything else is working, however empty body is a common case in spa world 😞

**Comment by vladshcherbin** [comment]
I believe the reason is here - https://github.com/posthtml/posthtml-insert-at/blob/0.2.4/src/insertNode.ts#L6, an empty tag has content `[ undefined ]` so nothing is processed.

**Comment by vladshcherbin** [comment]
As a current workaround, previous version **0.2.3** is working as expecting as it doesn't have this check.

**Comment by metonym** [comment]
@vladshcherbin Thank you for raising this issue. This is a serious regression – will work on a fix.

**Comment by vladshcherbin** [comment]
@metonym I also found a test snapshot change here - https://github.com/posthtml/posthtml-insert-at/commit/84dedc291f85b089c21f2dff513e7439e74f86fb#diff-a144de241ea76cf22ecea0b1811cd0e6R35

Here is link to [test](https://github.com/posthtml/posthtml-insert-at/blob/0.2.4/src/tests/__fixtures__/testCases.ts#L72-L74) and its [snapshot]( https://github.com/posthtml/posthtml-insert-at/blob/0.2.4/src/tests/__snapshots__/insertAt.test.ts.snap#L38). As you can see, `<footer>` is also not inserted in `#content`.

Hopefully this will help you investigate this issue and solution :)

**Comment by vladshcherbin** [comment]
Thank you for the plugin and such a quick fix 🎉

**Comment by metonym** [comment]
@vladshcherbin Thanks again for opening the issue and providing thorough documentation.

The bug is patched and the snapshot is updated.

Released in v0.2.5
